docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc

```diff
@@ -830,3 +831,129 @@ Error: error loading agent config: error loading raw config: fail to read config
 
 To resolve this, either install {agent} without the `--unprivileged` flag so that it has administrative access, or run the {agent} commands without the `sudo` prefix.
 
+[discrete]
+[[agent-kubernetes-kustomize]]
+== Troubleshoot {agent} installation on Kubernetes, with Kustomize
+
+Potential issues during {agent} installation on Kubernetes can be categorized into two main areas:
+
+. <<agent-kustomize-manifest>>.
+. <<agent-kustomize-after>>.
+
+[discrete]
+[[agent-kustomize-manifest]]
+=== Problems related to the creation of objects within the manifest
```

Comment:
```suggestion
=== Problems related to the creation of resources within the manifest
```

nit: Maybe resources sounds more k8s friendly?

Reply:
I have always considered and and referred to them as `kubernetes resources` too, and I see they are widely referenced as `resources`.

But some time ago, when looking at the official docs they are actually presented as objects: https://kubernetes.io/docs/concepts/overview/working-with-objects/

Also in these current blog posts the authors show a clear distinction:
- https://devopscube.com/kubernetes-objects-resources/
- https://iximiuz.com/en/posts/kubernetes-api-structure-and-terminology/

So, I think the right way to call them is `objects` (although it still feels strange to me).

Anyway, happy to change it to `resources` if you prefer. This is a topic I like, because in all the training I've been delivering for years I've always talked about k8s resources, and now I'm trying to talk about `objects` when referring to the persistent entities that represent the state of the cluster / records of intent (like `Pods`, `Deployments`, etc).

In summary:
- IMO a manifest uses and defines API resources that end up creating objects.
- A user should take care and check the objects related with the manifests they applied.

Reply:
You covered all my concerns :) Thanks for the detailed reply. I truly learned a lot here